lib/sync-DataSetModel.js

```diff
@@ -307,19 +307,19 @@ var self = {
     }
   },
 
-  getDatasetClientRecords: function (datasetClient, cb) {
-    var datasetClientId = datasetClient.id;
-    if (typeof self.datasetClientRecords[datasetClientId] !== "undefined") {
-      return cb(null, self.datasetClientRecords[datasetClientId]);
-    } else {
-      var datasetId = datasetClient.datasetId;
-      syncUtil.doLog(datasetId, 'silly', 'no data records found for datasetClient ' + datasetClientId + '. Do backend list.');
-      self.syncDatasetClient(datasetClient, function (err) {
-        if (err) return cb(err);
-        return cb(null, self.datasetClientRecords[datasetClientId]);
-      });
-    }
-  },
+  // getDatasetClientRecords: function (datasetClient, cb) {
+  //   var datasetClientId = datasetClient.id;
+  //   if (typeof self.datasetClientRecords[datasetClientId] !== "undefined") {
+  //     return cb(null, self.datasetClientRecords[datasetClientId]);
+  //   } else {
+  //     var datasetId = datasetClient.datasetId;
+  //     syncUtil.doLog(datasetId, 'silly', 'no data records found for datasetClient ' + datasetClientId + '. Do backend list.');
+  //     self.syncDatasetClient(datasetClient, function (err) {
+  //       if (err) return cb(err);
+  //       return cb(null, self.datasetClientRecords[datasetClientId]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)